_config.php

```diff
@@ -1,3 +1,3 @@
 <?php
 
-define('BOOTSTRAP_FORMS_DIR',basename(dirname(__FILE__)));
+define('BOOTSTRAP_FORMS_DIR','vendor/unclecheese/bootstrap-forms/');
```

Comment:
Don't need this defined anymore. Use module references in `Requirements::` calls.

Reply:
Fixed in https://github.com/unclecheese/silverstripe-bootstrap-forms/pull/56/commits/034d9e8b441a32fa13ca2271f0ec04a0e6c71525